### PR TITLE
Fix broken content form

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

**Comment by itkg-nanne** [comment]
:+1:

File: BackofficeBundle/Controller/ContentController.php [hunk 1]
```diff
@@ -57,6 +58,21 @@ public function formAction(Request $request, $contentId)
...
     /**
+     * Get the content language from Request or current site
+     * 
+     * @param Request $request
+     * 
+     * @return string
+     */
+    protected function getContentLanguage(Request $request)
+    {
+        if ($request->get('language'))
```
Comment (nicolasThal) [inline BackofficeBundle/Controller/ContentController.php]: or you can simply do $request->get('language', $this->get('open_orchestra.manager.current_site')->getCurrentSiteDefaultLanguage());

**Comment by itkg-ngilain** [comment]
@nicolasThal : (nice) update done

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**